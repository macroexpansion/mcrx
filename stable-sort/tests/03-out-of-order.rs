use stable_sort::stable_sorted;

#[stable_sorted]
pub enum Error {
    ThatFailed,
    ThisFailed,
    SomethingFailed,
    WhoKnowsWhatFailed,
}

fn main() {}
