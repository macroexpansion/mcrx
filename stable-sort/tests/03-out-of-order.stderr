error: SomethingFailed should sort before ThatFailed
 --> tests/03-out-of-order.rs:7:5
  |
7 |     SomethingFailed,
  |     ^^^^^^^^^^^^^^^
